docs/sdk/client-side/android/index.md

```diff
@@ -416,3 +418,33 @@ This method will return null if the feature flag is missing in the SDK.
 
 </TabItem>
 </Tabs>
+
+
+### Listening to evaluation updates
+
+BKTClient can notify when the evaluation is updated.  
```

Comment:
Please delete the space at the end of this line.

Reply:
these 2 spaces are needed to create a newline.